base/linalg/triangular.jl

```diff
@@ -976,18 +976,110 @@ for (f, g) in ((:/, :A_rdiv_B!), (:A_rdiv_Bc, :A_rdiv_Bc!), (:A_rdiv_Bt, :A_rdiv
     end
 end
 
+function ^(A::UpperTriangular, p::Integer)
```

Comment:
I think it would probably be clearer, and consistent with standard practice in linear algebra, to name your upper-triangular matrices `U` instead of `A`.